nautobot/core/api/urls.py

```diff
@@ -1,57 +1,29 @@
-from django.conf.urls import include
 from django.urls import path
-from drf_spectacular.views import (
-    SpectacularAPIView,
-    SpectacularJSONAPIView,
-    SpectacularYAMLAPIView,
-)
 
-from nautobot.core.api.views import (
-    APIRootView,
-    GetFilterSetFieldDOMElementAPIView,
-    GetFilterSetFieldLookupExpressionChoicesAPIView,
-    GraphQLDRFAPIView,
-    StatusView,
-    NautobotSpectacularSwaggerView,
-    NautobotSpectacularRedocView,
-)
-from nautobot.extras.plugins.urls import plugin_api_patterns
+from nautobot.core.api.views.dynamic_groups import DynamicGroupViewSet, DynamicGroupMembershipViewSet
+from . import routers, views
 
+router = routers.OrderedDefaultRouter()
+# router.APIRootView = views.CoreRootView
```

Comment:
Uncommenting this line (following a similar pattern for other Nautobot apps' `api.urls` code) results in an error from inside Django. I haven't chased it down.